src/components/RestaurantDetails.vue

```diff
@@ -29,48 +36,45 @@
       <h4>Linki</h4>
       <ul>
         <li>
-          <a :href="restaurant.url">{{restaurant.name}}</a><br/>
-          <a :href="restaurant.photos_url">Photos</a><br/>
-          <a :href="restaurant.menu_url">Menu</a><br/>
-          <a :href="restaurant.events_url">Events</a><br/>
+          <a :href="restaurant.url">{{ restaurant.name }}</a
+          ><br />
```

Comment:
Breaking a line line that (i.e. in the middle of a short tag closing) is one of the worse style choices one can make.